ledger/src/use_snapshot_archives_at_startup.rs

```diff
@@ -27,21 +27,17 @@ pub mod cli {
     pub const NAME: &str = "use_snapshot_archives_at_startup";
     pub const LONG_ARG: &str = "use-snapshot-archives-at-startup";
     pub const HELP: &str = "When should snapshot archives be used at startup?";
-    pub const LONG_HELP: &str = "At startup, when should snapshot archives be extracted \
-        versus using what is already on disk? \
-        \nSpecifying \"always\" will always startup by extracting snapshot archives \
```

Comment:
@brooksprumo - Is it correct to assume that you prefer the old formatting (which looks like it had some thought go into it) instead of what the `fmt` automatically did ? I can't remember where everyone landed the last time we had one of these reviews in regards to hand-crafted formatting vs. what `fmt` automatically gives us